docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_automanage/_index.md

```diff
@@ -0,0 +1,134 @@
+
+* Create Azure service principal (SP)
+
+    To connect a VM or bare-metal server to Azure Arc, Azure service principal assigned with the "Contributor" role is required. To create it, login to your Azure account run the below command (this can also be done in [Azure Cloud Shell](https://shell.azure.com/)).
+
+    ```shell
+    az login
+    az ad sp create-for-rbac -n "<Unique SP Name>" --role contributor
+    ```
+
+    For example:
+
+    ```shell
+    az ad sp create-for-rbac -n "http://AzureArcServers" --role contributor
+    ```
+
+    Output should look like this:
+
+    ```json
+    {
+    "appId": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "displayName": "AzureArcServers",
+    "name": "http://AzureArcServers",
+    "password": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "tenant": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+    }
+    ```
+
+  > **Note: It is optional but highly recommended to scope the SP to a specific [Azure subscription and resource group](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest). To create an Automanage Account used by the Automanage services, you need the Owner or Contributor permissions on your subscription along with User Access Administrator roles.**
+
+## Automation Flow
+
+For you to get familiar with the automation and deployment flow, below is an explanation.
+
+* User is editing the shell script to provide environment variables (1-time edit). These variables are being used throughout the deployment.
+
+* The script will edit the ARM templates parameters file that will be used for the deployment. There are three ARM templates that will:
+
+  * Create an Azure Automanage Account.
+  * Grant premissions to the Azure Automanage Accout.
+  * Enable Azure Automanage on the Azure Arc enabled server.
+
+## Enable Azure Automanage on an Azure Arc enabled server
+
+* In order to keep your local environment clean and untouched, we will use Azure Cloud Shell to run the `arc_automanage.sh` shell script against the Azure Arc enabled server.
+
+  ![Screenshot showing Azure Cloud Shell](./03.png)
+
+* To run the automation, navigate to the [deployment folder](https://github.com/microsoft/azure_arc/tree/main/azure_arc_servers_jumpstart/automanage/artifacts) and edit [the script's](https://github.com/microsoft/azure_arc/tree/main/azure_arc_servers_jumpstart/automanage/artifacts/arc_automanage.sh) environment variables, you have to provide:
+  * **automanageAccountName:** name of your Automanage Account identity, make sure it is unique.
+  * **location:** Azure Region where your Arc enabled server is registered to.
+  * **machineName:** Name of your Azure Arc enabled server as it is shown in the Azure Portal.
+  * **configurationProfileAssignment:** refers to the environment of your Azure Arc enabled server as Azure Automanage has different profiles. Values can be "Production" or "DevTest".
+
+ ![Screenshot environment variables](./04.png)
```

Comment:
See command on parameter name _resourceGroup_

Reply:
modified to resource_group